modules/swagger-codegen/src/main/java/io/swagger/codegen/examples/ExampleGenerator.java

```diff
@@ -106,6 +107,8 @@ protected Object resolvePropertyToExample(String mediaType, Property property, S
             return 1.23f;
         } else if (property instanceof IntegerProperty) {
             return 123;
+        } else if (property.getClass().equals(BaseIntegerProperty.class)) {
```

Comment:
If we use `instanceof` it will return true for `LongProperty` and `IntegerProperty` which inherit from `BaseIntegerProperty`. I don't know if it matters specifically for nodejs-server, but swagger-parser makes the distinction and this change affects all languages. Ints without a format are passed from swagger-parser as BaseInetgerProperties and nothing more.